### DISCUSSION TITLE
Snippets!

### DISCUSSION THREAD

**Discussion by felixjb** [felixjb/testify#64] [General]
I like very much to use snippets for testing, but the extension and snippets I currently have are not as customizable and useful as I think they could be. 

I think maybe I could add this feature to Testify (there's an open issue for it #13), but maybe it would be too much for the project.

What do **you** guys think?

**Comment by felixjb** [comment]
@calebboyd @roggenbrot @silvawillian you guys are contributors. What do you think?

**Reply by roggenbrot** [reply]
Sorry, was busy with other projects/languages in the past year and wasn't working a lot with jest

Snippets are nice, but I think there are a other features which would improve the extensions even more. 

While working with Intellij and Java I learned to love the codelens in IntelliJ that is displayed next to failed tests and makes it easy just to reun such failed tests manually. 

Such a feature in vs-code would be awesome.
